hw_diag/tests/test_get_public_keys.py

```diff
@@ -6,24 +6,21 @@
 
 
 class TestGetPublicKeys(unittest.TestCase):
-    TEST_DATA = [
-        "{pubkey,",
-        '"112mPWkGW55kcbQTgbtJvgAKMSTrEhHgavrdF1Cbu8FU85tTL4Nc"}.',
-        '{onboarding_key,',
-        '"112mPWkGW55kcbQTgbtJvgAKMSTrEhHgavrdF1Cbu8FU85tTL4Nc"}.',
-        '{animal_name,',
-        '"gigantic-basil-turtle"}.'
-    ]
+    TEST_DATA = '{}\n{}\n{}'.format(
```

Comment:
I'm unsure about the line breaks. The miner I'm looking at doesn't appear to have any.

Reply:
It was introduced in the latest GA. We've had half the fleet fall over because of this.

Reply:
```
/tmp/build # wc -l /var/data/public_keys
3 /var/data/public_keys
```